packages/components/src/tree-grid/stories/index.tsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import TreeGrid, { TreeGridRow, TreeGridCell } from '..';
+import { Button } from '../../button';
+
+const meta: ComponentMeta< typeof TreeGrid > = {
+	title: 'Components (Experimental)/TreeGrid',
+	component: TreeGrid,
+	subcomponents: { TreeGridRow, TreeGridCell },
```

Comment:
I am hesitant to add `TreeGridItem` to the subcomponents table because:

- It wasn't even documented in the main readme.
- It seems quite low level, and I'm not sure if the few usages in the Gutenberg app are even warranted in the first place 🤔 
- The docgen refuses to generate anything, probably because the types are too loose. (It needs to accept any prop, as they are passed through to arbitrary `children`/`as`.)

Reply:
Makes sense. Let's keep it out of the docs, and probably prepare a follow-up to see if we can fix those `TreeGridItem` usages and convert them to `TreeGridCell`